content/kubermatic/main/architecture/concept/kkp-concepts/kkp-security/iam/_index.en.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Kubermatic Kubernetes Platform IAM"
+date = 2023-02-07T12:15:02+02:00
+weight = 20
+
++++
+
+KKP component (dashboard, user clusters ...) use OpenID Connect (OIDC) protocol for authentication and RBAC for authorization.
```

Comment:
```suggestion
KKP components (dashboard, user clusters ...) use the OpenID Connect (OIDC) protocol for authentication and RBAC for authorization.
```

In addition, I think we need to clarify this sentence and highlight that this RBAC is _not_ Kubernetes RBAC but a KKP-specific implementation, at least when it comes to interacting with KKP itself. I know this is explained later, but grouping the dashboard and user clusters is a bit misleading here to be honest.